src/panels/lovelace/editor/config-elements/hui-heading-card-editor.ts

```diff
@@ -56,7 +62,7 @@ export class HuiHeadingCardEditor
 
   public setConfig(config: HeadingCardConfig): void {
     assert(config, cardConfigStruct);
-    this._config = config;
+    this._config = migrateHeadingCardConfig(config);
```

Comment:
If you do the migration before the assert we can remove entities from the struct